docs/modules/ROOT/pages/set-budget.adoc

```diff
@@ -0,0 +1,37 @@
+= Setting a Budget
+:description: Use the Budget Tracker to help you to monitor and control your spending in Hazelcast Viridian.
+:cloud-tags: Manage Accounts
+:cloud-title: Setting a Budget
+:cloud-order: 58
+
+{description}
+
+Set a budget and you'll receive an email alert when your costs reach 80% of your budget, and again when you hit your budget. You can use this information to take action to limit your spending or to gain insights into your resource usage.
```

Comment:
Please note, a customer will receive an email as well as notification warning in Viridian

Reply:
@alaktionov, where does the notification appear?

Reply:
It appears at the top of the page and is persistent across all pages unless you update your Daily/Monthly Budget Alert
![image](https://user-images.githubusercontent.com/6086036/212693003-824e1ba1-f155-421a-b801-f4a4c0b141c4.png)

Reply:
Added to instructions.